containerimages/container-disk-images/build-container-disk.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -ex
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$BASH_SOURCE[0]")"
+    pwd
+)"
+
+image_name=$1
+tag=$2
+vm_image_file=$3
+
+export CONTAINER_DISK_DOCKERFILE=${CONTAINER_DISK_DOCKERFILE:-$SCRIPT_DIR/Dockerfile.template}
+readonly IMAGE_PLACEHOLDER="IMAGE"
+
+sed s?$IMAGE_PLACEHOLDER?"${vm_image_file}"?g "$CONTAINER_DISK_DOCKERFILE" > "Dockerfile"
```

Comment:
maybe will be nice to use envsubst?

Reply:
Yes it is better then using `sed`, I will change it in following PR at kubevirtci as discussed in last community meeting.